src/json.h

```diff
@@ -0,0 +1,24 @@
+/** @file
```

Comment:
Can we follow the style of heading for new files as seen in others? Feel free to keep your name and date (maybe add an e-mail if you feel comfortable enough). I just ask that we make all file headings look consistent.

Reply:
Fixed!